### DISCUSSION TITLE
I edited plotman.yalm and need to reload

### DISCUSSION THREAD

**Discussion by rodrigo081089** [ericaltendorf/plotman#342] [Q&A] [answered]
I edited plotman.yalm  to add jobs but plotman does not reload it. How can i use the new configuration?

Thanks

**Comment by BasilHorowt** [comment, marked as answer]
The configuration is parsed as a plotman command is launched. So if you are running plotman interactive when you change the config file, you'll have to q to quit and re-run plotman interactive to reflect any changes.

Check out the FAQ for some other useful info: https://github.com/ericaltendorf/plotman/wiki/FAQ#how-do-you-reload-the-plotman-config-file
